src/components/Proposal/ActionButton.tsx

```diff
@@ -144,43 +139,43 @@ class ActionButton extends React.Component<IProps, IState> {
         {this.state.preRedeemModalOpen ?
           <PreTransactionModal
             actionType={executable && !redeemable ? ActionTypes.Execute : ActionTypes.Redeem}
-            action={redeemProposal.bind(null, dao.address, proposal.id, proposal.contributionReward.beneficiary)}
+            action={redeemProposal.bind(null, dao.address, proposalState.id, currentAccountAddress)}
             beneficiaryProfile={beneficiaryProfile}
             closeAction={this.closePreRedeemModal.bind(this)}
             dao={dao}
             effectText={redemptionsTip}
-            proposal={proposal}
+            proposal={proposalState}
           /> : ""
         }
 
-        { proposal.stage === IProposalStage.Queued && proposal.upstakeNeededToPreBoost.lte(new BN(0)) ?
+        { proposalState.stage === IProposalStage.Queued && proposalState.upstakeNeededToPreBoost.lte(new BN(0)) ?
             <button className={css.preboostButton} onClick={this.handleClickExecute.bind(this)} data-test-id="buttonBoost">
               <img src="/assets/images/Icon/boost.svg"/>
               { /* space after <span> is there on purpose */ }
               <span> Pre-Boost</span>
             </button> :
-          proposal.stage === IProposalStage.PreBoosted && expired && proposal.downStakeNeededToQueue.lte(new BN(0)) ?
+          proposalState.stage === IProposalStage.PreBoosted && expired && proposalState.downStakeNeededToQueue.lte(new BN(0)) ?
             <button className={css.unboostButton} onClick={this.handleClickExecute.bind(this)} data-test-id="buttonBoost">
               <img src="/assets/images/Icon/boost.svg"/>
               <span> Un-Boost</span>
             </button> :
-          proposal.stage === IProposalStage.PreBoosted && expired ?
+          proposalState.stage === IProposalStage.PreBoosted && expired ?
             <button className={css.boostButton} onClick={this.handleClickExecute.bind(this)} data-test-id="buttonBoost">
               <img src="/assets/images/Icon/boost.svg"/>
               <span> Boost</span>
             </button> :
-          (proposal.stage === IProposalStage.Boosted || proposal.stage === IProposalStage.QuietEndingPeriod) && expired ?
+          (proposalState.stage === IProposalStage.Boosted || proposalState.stage === IProposalStage.QuietEndingPeriod) && expired ?
             <button className={css.executeButton} onClick={this.handleClickExecute.bind(this)}>
               <img src="/assets/images/Icon/execute.svg"/>
               { /* space after <span> is there on purpose */ }
               <span> Execute</span>
             </button>
           : redeemable ?
             <div>
-              { !detailView ?
-                  <RedemptionsString currentAccountAddress={currentAccountAddress} dao={dao} proposal={proposal} rewards={rewardsForCurrentUser} />
+              {/* !detailView ?
```

Comment:
we dont want this? or its just broken for now?

Reply:
quoting from the PR description: "remove the redemptionsstring (next to the redeem button) as it is was showing the wrong results and it is unclear what is should show"

I thiknk we are making a mistake showing similar info in different formats in different places in slightly different ways, and we should rethink that approach.